setup.py

```diff
@@ -20,6 +20,11 @@
         'Operating System :: OS Independent',
         'Programming Language :: Python',
         'Framework :: Django',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
```

Comment:
3.5 is not in the tox testing matrix.

Reply:
Just added 3.5 to tox testing matrix, thanks.